src/types.rs

```diff
@@ -1207,11 +1207,7 @@ pub unsafe fn isascii(mut _c: libc::c_int) -> libc::c_int {
 
 #[inline]
 pub unsafe fn isspace(mut _c: libc::c_int) -> libc::c_int {
-    if _c < std::u8::MAX as libc::c_int {
-        ((_c as u8 as char) == ' ') as libc::c_int
-    } else {
-        0
-    }
+    libc::isspace(_c)
```

Comment:
This fixes tests for `dc_trim()`, `dc_ltrim()`, `dc_rtrim()` and `dc_simplify_simplify()`.

Reply:
sweet, no need to wrap it though, you can just import the method directly and delete this method

Reply:
Sure. I just thought this was the easiest since there were a bunch of them. Will fix.